src/components/info/CalenderInfo.tsx

```diff
@@ -1,11 +1,62 @@
 import React from 'react';
-import { View } from 'react-native';
-import { Text } from 'react-native-paper';
+import { View, Image, StyleSheet } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import { Text, Headline, Subheading, Paragraph } from 'react-native-paper';
 
 export const CalenderInfo = () => {
```

Comment:
```suggestion
export const CalendarInfo = () => {
```
The file name also needs changing